public/styles/main.css

```diff
@@ -333,6 +333,54 @@ a.connections-table.icon {
   width: 12px;
   background-image: url('../images/info-accent.svg');
 }
+.category-container {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows: 1fr 1fr;
```

Comment:
I don't think we need to force a 2x2 grid here. Have a look at `repeat` and `auto-fit` in particular